Core/MSBuild/MSBuild.cs

```diff
@@ -51,17 +51,17 @@ public void CallMSBuild (string msBuildPath, string arguments)
     msBuildStartInfo.WorkingDirectory = Environment.CurrentDirectory;
 
     var msBuildProcess = Process.Start(msBuildStartInfo);
+
+    var msBuildOutput = msBuildProcess!.StandardOutput.ReadToEnd();
+    
     msBuildProcess!.WaitForExit(c_msBuildProcessTimeout);
 
     if (msBuildProcess.ExitCode != 0)
     {
-      var errorMessage = msBuildProcess.StandardError.ReadToEnd();
-      var errorOutput = msBuildProcess.StandardOutput.ReadToEnd();
-      var message = $"MSBuild '{arguments}' failed with Error: '{errorOutput}'.";
+      var message = $"MSBuild '{arguments}' failed with Error: '{msBuildOutput}'.";
```

Comment:
If we are not using the error stream we should do `RedirectStandardError = false` (or in this case remove the line)